src/components/AccessControl.vue

```diff
@@ -155,7 +155,8 @@ export default {
         <breadcrumb
           :location="model"
           readonly="readonly"
-          no-root="no-root"/>
+          no-root="no-root"
```

Comment:
I was hoping there was some rule that would change these boolean properties back to just the `no-root` form. This is technically equivalent, just seems less preferable than the shorthand.

Reply:
Created #242 because I don't want to leave it like this, but I don't feel like fixing it right now.